ledger/participant-integration-api/src/test/suite/scala/platform/store/backend/StorageBackendPostgresInitializationSpec.scala

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.platform.store.backend
+
+import com.daml.ledger.api.domain.{LedgerId, ParticipantId}
+import com.daml.lf.data.Ref
+import com.daml.platform.common.MismatchException
+import org.scalatest.flatspec.AsyncFlatSpec
+import org.scalatest.matchers.should.Matchers
+
+final class StorageBackendPostgresInitializationSpec
+    extends AsyncFlatSpec
+    with StorageBackendPostgresSpec
```

Comment:
I think this needs some further improvements here: we need a framework to have tests declared only once, and executed for all backends (H2, Postgres, Oracle, similar to JdbcLedgerDao tests).

Reply:
This was just a quick implementation to have _some_ unit tests. PR https://github.com/digital-asset/daml/pull/10529 adds a whole framework for running the tests on each backend.